### ISSUE TITLE
Add an option for Muse Sounds to allow playback of notes beyond playable range

### ISSUE THREAD

**Issue by overcast07** [musescore/MuseScore#20145]
### Your idea

There should be a toggle somewhere in the MuseScore app that changes the behavior of Muse Sounds, such that notes of any pitch outside the playable range of an instrument can be played during playback.

The notes would be procedurally generated using the highest or lowest defined sample of the instrument as appropriate.

(An alternative solution would be to make this the default behavior, which would obviate the need for the toggle.)

### Problem to be solved

Based on #16244 and #13894, it seems that there has been an intentional choice to limit the playback range of Muse Sounds to the actual playable range of instruments.

In some contexts, such as transcribing a specific track that was produced digitally, it may make sense to use a note that is outside of the playable range and also play it during playback. The use of notes outside the playable range of a corresponding real instrument is probably fairly common in some contexts such as contemporary soundtracks that use instrument libraries to mimic specific real instruments, and sequenced video game music from the 1990s and 2000s that uses samples of real instruments. Furthermore, it would be unhelpful to notate every instrument in such a score as a synthesizer, even in cases where no live instruments were used.

As an example, earlier I was transcribing a part of "[Beware of Forest Mushrooms](https://www.youtube.com/watch?v=PMVqFUCBbkE)" from _Super Mario RPG_ (2023). I did this in order to produce a score, a MIDI file and an audio file. The oboe instrument in this track plays several notes that are below the playable range of the real instrument, which Muse Sounds currently refuses to play. In order to create an audio file with Muse Sounds instruments at the correct pitch, I was forced to transpose the notes up 5 semitones before exporting, and then transpose the audio back down again using FL Studio. (I chose to do this instead of using the MuseScore 3 soundfont because of the difference in instrument quality.)

<img width="950" alt="Beware of Forest Mushrooms" src="https://github.com/musescore/MuseScore/assets/60053886/8c4341d6-d698-4f72-969f-b64e4042ae76">

This hopefully demonstrates that there is an actual use case for arbitrarily extending the playback range of any pitched instrument in Muse Sounds, even if this would not be a desirable default behavior.

**Comment by s11141827** [comment]
That would be a cool upgrade, in fact the Oboe D'amore (aka Mezzo-Soprano Oboe) goes below the range of a normal oboe

**Comment by s11141827** [comment]
We should also extend the Tuba (& Cimbasso Range too) lower than E1 because they often have 4 (or more) Valves to allow for a larger note range. They even make 4 valve trumpets, etc.

**Comment by overcast07** [comment]
I should clarify that the scope of this issue wasn't supposed to only encompass instruments which have siblings with different pitch ranges. That might warrant its own separate issue, but there is no need to list off instrument variants in the comments here. It is not relevant.

The proposal is specifically to arbitrarily allow notes to be played at **any** pitch, even if that pitch isn't audible by humans, to allow for accurate transcription of tracks that happen to include those notes for whatever reason.

**Comment by s11141827** [comment]
> I should clarify that the scope of this issue wasn't supposed to only encompass instruments which have siblings with different pitch ranges. That might warrant its own separate issue, but there is no need to list off instrument variants in the comments here. It is not relevant.
> 
> The proposal is specifically to arbitrarily allow notes to be played at **any** pitch, even if that pitch isn't audible by humans, to allow for playback of accurate transcriptions of tracks that happen to include those notes for whatever reason.

That would be as COOL AS COCONUTS.

**Comment by s11141827** [comment]
I changed the comment up a bit so it would be just that "That would be a cool upgrade".

**Comment by s11141827** [comment]
BTW The highest note on the Harpsichord in Musesounds is Bb5 (lowest note is Bb1) so we need to broaden the range soon enough so I could get the full range as Harpsichords generally have a larger note range than 4 octaves.

**Comment by DaddyLudwig** [comment]
> BTW The highest note on the Harpsichord in Musesounds is Bb5 (lowest note is Bb1) so we need to broaden the range soon enough so I could get the full range as Harpsichords generally have a larger note range than 4 octaves.

Exactly, I have to resort to soundfonts for Bach harpsichord concertos and Brandenburg 5 because Bach goes up to D6 and Muse Sounds doesn’t have D6.
